src/containers/shared/metaParser.tsx

```diff
@@ -106,6 +126,9 @@ export function findNodes(meta: any, entryType: string) {
   return meta.AffectedNodes.filter(
     (node: any) =>
       node.CreatedNode?.LedgerEntryType === entryType ||
-      node.ModifiedNode?.LedgerEntryType === entryType,
```

Comment:
This method should be refactored to add a `type` key or something so that you know if it's a created, modified, or deleted node.

Reply:
we dont know the type of the node we need to access - i think this is a clean way to handle this.

Reply:
We don't know the type, yes - but we might need to know the type later. So if we keep track of which type of node it is that has this data, it'll simplify later processing.